src/i18n/locale/en.json

```
@@ -453,6 +453,24 @@
   "components.Settings.SettingsJobsCache.runnow": "Run Now",
   "components.Settings.SettingsJobsCache.sonarr-scan": "Sonarr Scan",
   "components.Settings.SettingsJobsCache.unknownJob": "Unknown Job",
+  "components.Settings.SettingsLogs.filterDebug": "Debug",
+  "components.Settings.SettingsLogs.filterError": "Error",
+  "components.Settings.SettingsLogs.filterInfo": "Info",
+  "components.Settings.SettingsLogs.filterWarn": "Warning",
+  "components.Settings.SettingsLogs.label": "Label",
+  "components.Settings.SettingsLogs.level": "Severity",
+  "components.Settings.SettingsLogs.logs": "Logs",
+  "components.Settings.SettingsLogs.logsDescription": "You can also view these logs directly via <code>stdout</code>, or in <code>{configDir}/logs/overseerr.log</code>",
```

```suggestion
  "components.Settings.SettingsLogs.logsDescription": "You can also view these logs directly via <code>stdout</code>, or in <code>{configDir}/logs/overseerr.log</code>.",
```
Missing a period at the end of this sentence 😸